src/engine/strat_engine/pool.rs

```diff
@@ -199,6 +199,21 @@ impl StratPool {
             }
         }
     }
+
+    /// Teardown a pool.
+    /// Take down the device mapper devices belonging to the pool.
+    /// This method and destroy() must keep their DM teardown operations
+    /// in sync.
+    pub fn teardown(self) -> EngineResult<()> {
+        // TODO: sync files
```

Comment:
Not sure -- we may find all filesystems need to be unmounted to teardown, not just synced.